editor/src/components/editor/actions/actions.ts

```diff
@@ -1232,7 +1241,7 @@ function replaceFilePath(
   if (error == null) {
     return {
       type: 'SUCCESS',
-      projectContents: updatedProjectContents,
+      projectContents: contentsToTree(updatedProjectContents),
```

Comment:
Please can you either add some `// FIXME`s around functions like this (that convert between the old and new type to ease our transition), since we'll definitely want to replace those

Reply:
Can do.

Reply:
Fixed in 477c7b28.